### PR TITLE
[Avatar Addition] Sworn Public

### PR DESCRIPTION
**Pull request by Vexprimagen2**

### CODE REVIEW

**Comment by Happyrobot33** [comment]
Can you provide some information on this addition? Is this a personal base, something from gumroad?

File: Packages/com.mattshark.openflight/Runtime/data.json [hunk 1]
```diff
@@ -1177,6 +1178,28 @@
...
+        "Name": "Sworn Public",
+        "Creator": "Alıən",
+        "Introducer": "Vexprimagen",
+        "Hash": ["341832v2"],
+        "Weight": 1,
+        "WingtipOffset": 4.03  
+            
+      }
+     },
+      "Sworn Public": {
```
Comment (Happyrobot33) [inline Packages/com.mattshark.openflight/Runtime/data.json:1193]: key is duplicated twice

**Review by Happyrobot33 — CHANGES_REQUESTED** [review]

**Comment by Vexprimagen2** [comment]
It was an avatar that was brought to me that was not in the open flight system that had wings and I asked another contributor if it could be hashed and I was told that it should be fine

**Comment by Happyrobot33** [comment]
> It was an avatar that was brought to me that was not in the open flight system that had wings and I asked another contributor if it could be hashed and I was told that it should be fine
> 

I really need more information about the base than this. Even attaching a picture would help

**Comment by Happyrobot33** [comment]
Sorry, I cannot accept this PR, as the avatar trying to be added matches the nardoragon hash. The hash you submitted is `341832v2` and you can see in this [list](https://github.com/Mattshark89/OpenFlight-VRC/blob/85f882b85246a9e3982e3bc52194768337c5508a/Packages/com.mattshark.openflight/Runtime/data_schema.json#L55) that its the nardoragon hash

**Pull request closed without merging**